core/pkg/containerscan/elasticadapters.go

```diff
@@ -5,55 +5,7 @@ import (
 	cautils "github.com/armosec/utils-k8s-go/armometadata"
 )
 
-// ToFlatVulnerabilities - returnsgit p
-func (scanresult *ScanResultReport) ToFlatVulnerabilities() []*ElasticContainerVulnerabilityResult {
```

Comment:
I am surprised by this one... did you cover that case elsewhere and I didn't notice?

Reply:
I removed it because I couldn't find any function call and I thought it was obsolete.